packages/web-components/src/components/cbp-nav-item/cbp-nav-item.scss

```diff
@@ -1,68 +1,79 @@
+ * @Prop  --cbp-nav-item-color: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-color-dark: var(--cbp-color-interactive-secondary-lighter);
 
- * @Prop   --cbp-nav-item-button-color-hover: var(--cbp-color-interactive-secondary-darker);
- * @Prop   --cbp-nav-item-button-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-nav-item-color-hover: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
 
- * @Prop   --cbp-nav-item-button-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
- * @Prop   --cbp-nav-item-button-color-bg-hover-dark: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-nav-item-color-bg-hover-dark: var(--cbp-color-interactive-secondary-darker);
     
- * @Prop   --cbp-nav-item-button-border-bottom-hover: var(--cbp-color-interactive-secondary-darker);
- * @Prop   --cbp-nav-item-button-border-bottom-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+ * @Prop   --cbp-nav-item-border-bottom-hover: var(--cbp-color-interactive-secondary-darker);
+ * @Prop   --cbp-nav-item-border-bottom-hover-dark: var(--cbp-color-interactive-secondary-lighter);
 
  */
 
  :root {
-    --cbp-nav-item-button-color: var(--cbp-color-interactive-secondary-darker);
-    --cbp-nav-item-button-color-dark: var(--cbp-color-interactive-secondary-lighter);
+  --cbp-nav-item-color: var(--cbp-color-interactive-secondary-darker);
+  --cbp-nav-item-color-dark: var(--cbp-color-interactive-secondary-lighter);
 
-    --cbp-nav-item-button-color-hover: var(--cbp-color-interactive-secondary-darker);
-    --cbp-nav-item-button-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+  --cbp-nav-item-color-hover: var(--cbp-color-interactive-secondary-darker);
+  --cbp-nav-item-color-hover-dark: var(--cbp-color-interactive-secondary-lighter);
 
-    --cbp-nav-item-button-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
-    --cbp-nav-item-button-color-bg-hover-dark: var(--cbp-color-interactive-secondary-darker);
-    
-    --cbp-nav-item-button-border-bottom-hover: var(--cbp-color-interactive-secondary-darker);
-    --cbp-nav-item-button-border-bottom-hover-dark: var(--cbp-color-interactive-secondary-lighter);
+  --cbp-nav-item-color-bg-hover: var(--cbp-color-interactive-secondary-lighter);
+  --cbp-nav-item-color-bg-hover-dark: var(--cbp-color-interactive-secondary-darker);
+  
+  --cbp-nav-item-border-bottom-hover: var(--cbp-color-interactive-secondary-darker);
+  --cbp-nav-item-border-bottom-hover-dark: var(--cbp-color-interactive-secondary-lighter);
 
 }
 
 /* 
- * Dark Mode - display dark design based on mode or context.
- */
- 
+* Dark Mode - display dark design based on mode or context.
+*/
+
 [data-cbp-theme=light] cbp-nav-item[context*=dark]:not(#fakeId),
```

Comment:
With a parent-child relationship, I think it makes more sense to look at the context of the parent. These children don't even need a context prop really. I think you can just reference the parent component here instead of the nav-item and have it work the same (toggling nav-item specific variables)

Reply:
updated selector to use the parent for context